src/components/scenes/SettingsOverviewScene.js

```diff
@@ -86,6 +87,33 @@ export default class SettingsOverview extends Component<Props> {
     }
   }
 
+  async componentDidMount () {
+    if (!this.props.supportsTouchId) {
+      return null
+    }
+    try {
+      const biometryType = await getSupportedBiometryType()
+      switch (biometryType) {
+        case 'FaceID':
+          this.options.useTouchID.text = s.strings.settings_button_use_faceID
+          this.forceUpdate()
```

Comment:
Do not use forceUpdate(). Instead, move this.options.useTouchID.text to component State to it so that you automatically re-render when it changes

Reply:
Fix on the new commit